language/bytecode-verifier/transactional-tests/tests/control_flow/invalid_fallthrough3.exp

```diff
@@ -1,10 +1,10 @@
 processed 1 task
 
-task 0 'run'. lines 1-12:
+task 0 'run'. lines 1-13:
 Error: Script execution failed with VMError: {
     major_status: INVALID_FALL_THROUGH,
     sub_status: None,
     location: script,
     indices: [],
-    offsets: [(FunctionDefinitionIndex(0), 5)],
+    offsets: [(FunctionDefinitionIndex(0), 6)],
```

Comment:
bit strange the offsets changed here

Reply:
The difference is this: before, `if (condition) { /* true_block */ } else { /* false_block */ }` would result in the bytecode for the statements in `true_block` being placed immediately after the bytecode for the `condition` expression. So, the branching behavior could be expressed in just a single bytecode instruction: `BrFalse` --  that is, "branch to the `false_block` if the condition is false, or fallthrough to the `true_block` otherwise."

With this change, the statements for the `true_block` can be placed anywhere and in any order:

```
f() {
    let condition: bool;
    condition = true;
    if (condition) b0 else b1;
label b1:
    abort 1;
label b0:
    return;
}
```

So MVIR can no longer express this behavior with just a single instruction, it needs two: `BrTrue` to the `true_block` if `condition` is true, otherwise `Branch` to the `false_block`.